pyproject.toml

```diff
@@ -13,61 +13,61 @@ keywords = ["sed", "mpes", "flash", "arpes"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.8, <3.12"
 bokeh = "^2.4.2"
-dask = ">=2021.12.0, <2023.0.0"
-fastdtw = "^0.3.4"
-fastparquet = "^0.8.0"
-h5py = "^3.6.0"
+dask = ">=2021.12.0"
+fastdtw = ">=0.3.4"
+fastparquet = ">=0.8.0"
+h5py = ">=3.6.0"
 ipympl = ">=0.9.1"
 ipywidgets = ">=7.7.1"
-lmfit = "^1.0.3"
-matplotlib = "^3.5.1"
-natsort = "^8.1.0"
-numba = "^0.55.1"
-numpy = ">=1.18,<1.23"
+lmfit = ">=1.0.3"
```

Comment:
Is there a reason to changing `^` to `>=` ?

Poetry specification recommends https://python-poetry.org/docs/dependency-specification/ and also when we do poetry add <package-name>, it automatically uses `^`

Reply:
Using ^ will prevent any update of the highest version in the hierarchy, e.g. and update from 0.x.y to 0.x+1.y will be prevented. While this might be a good idea for packages following the major.minor.patch versioning style, many packages have adopted a year.month.day style of versioning, where ^essentially suppresses most updates, at least ones in a new year.
As many packages needed update for python 3.11, I essentially opted to remove any upper limits. I think we have good enough tests to recognize any incompatibilities, allowing us to add respective limits in time.